saleor/graphql/shop/types.py

```diff
@@ -76,6 +77,8 @@ class Shop(graphene.ObjectType):
         required=True)
     default_currency = graphene.String(
         description='Default shop\'s currency.', required=True)
+    available_payment_gateways = graphene.List(
```

Comment:
This field will never be `null` so we should add `required=True`.

Reply:
Well, theoretically it would make sense, but did we really had any use case in any of our storefronts for that?

Reply:
Each gateway has its limitations: some only accept cards issued in certain countries, most of them are not available in China, etc.

Reply:
OK, we could move this field to the `Checkout` type. We could keep the resolver to return gateways that have been configured in settings, but later this will be a good place to add some logic.

Reply:
Done